src/sentry/static/sentry/app/views/alerts/details/body.tsx

```diff
@@ -57,11 +58,17 @@ export default class DetailsBody extends React.Component<Props> {
         .filter(({slug}) => incident.projects.includes(slug))
         .map(({id}) => Number(id)),
       version: 2 as const,
-      range: `${incident.alertRule.timeWindow}m`,
+      start: incident.dateStarted,
+      end: incident.dateClosed ?? getUtcDateString(new Date()),
```

Comment:
This start/end range could be bad if it started very recently (e.g. 1 minute ago).

Reply:
@billyvg why is that bad? does that produce undesirable results?

Reply:
Bad as in you could have a start-end range of 1 minute with a time window of 1 minute, so you only chart 1 data point

Reply:
Oh I see.

Reply:
For that edge case, we could set `start` to be earlier; like 30minutes or so back.

Maybe this function could help in detecting tight chart windows. https://github.com/getsentry/sentry/blob/9a802f3d69f8bea59c834aaf7762861be90e1603/src/sentry/static/sentry/app/components/charts/utils.tsx#L65

Reply:
We're gonna do (start - timewindow - 20% of diff of start/end or now)

Reply:
Yea that works. 👍